src/app/AppFooter/AppFooter.js

```diff
@@ -1,23 +1,22 @@
 import styled from 'styled-components/macro'
 import Navigation from '../Navigation'
 
-export default function AppFooter({className}) {
-
+export default function AppFooter({ className }) {
   return (
-    <FooterStyled className={className}>
-      <Navigation/>
-    </FooterStyled>
+    <Footer className={className}>
+      <Navigation />
+    </Footer>
   )
 }
 
-const FooterStyled = styled.footer`
+const Footer = styled.footer`
+  height: 50px;
+  max-width: 1240px;
```

Comment:
Your `AppWrapper` has a `max-width: 600px`, I think the` AppFooter` should have the same or a smaller width.